src/network/rover_system_messages/control_control.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package control;
+
+message Main {
+    float ps_batt = 1;
+    float main_curr = 2;
+}
+
+message PS12 {
+	float ps12_volt = 1;
+    float ps12_curr = 2;
+    float temp12 = 3;
+}
+
+message PS5 {
+	float ps5_volt = 1;
+    float ps5_curr = 2;
+    float temp5 = 3;
+}
+
+message Odrv {
+    float odrv0_curr = 1;
+    float odrv1_curr = 2;
+    float odrv2_curr = 3;
+}
```

Comment:
I'd rename the package to `sensors` or similar since we aren't really controlling anything. Also the proto file name doesn't need to be prefixed with `control_`

Also maybe use more descriptive names (less or no abbreviations) for the message types and files since I don't immediately know what they mean